website/client/src/components/groupLink.vue

```diff
@@ -1,26 +1,36 @@
 <template>
-  <b-link
-    v-if="group"
```

Comment:
the existing `v-if` here should be kept?

Reply:
Good spot. I'll get that added back in.